api/src/main/java/org/pmiops/workbench/cloudtasks/TaskQueueService.java

```diff
@@ -245,28 +218,43 @@ public void pushInitialCreditsExhaustionTask(
 
   public void groupAndPushCheckInitialCreditExpirationTasks(List<Long> userIds) {
     WorkbenchConfig workbenchConfig = workbenchConfigProvider.get();
-    CloudTasksUtils.partitionList(
-            userIds, workbenchConfig.offlineBatch.usersPerCheckInitialCreditsExpirationTask)
-        .forEach(
-            batch ->
-                createAndPushTask(
-                    CHECK_CREDITS_EXPIRATION_FOR_USER_IDS_QUEUE_NAME,
-                    CHECK_CREDITS_EXPIRATION_FOR_USER_IDS_PATH,
-                    batch));
+    createAndPushAll(
+        userIds,
+        workbenchConfig.offlineBatch.usersPerCheckInitialCreditsExpirationTask,
+        INITIAL_CREDITS_EXPIRATION);
+  }
+
+  private TaskQueuePair withRdrBackfill(TaskQueuePair pair) {
+    return new TaskQueuePair(pair.queueName(), pair.endpoint() + "?backfill=true");
```

Comment:
Is `backfill` the only query parameter possible for a task queue endpoint? I wonder if TaskQueuePair could have an optional parameter (alternate constructor) that specified query parameters.

Reply:
These are just normal RWB POST endpoints, so they could look like anything that's valid.  I'm not currently anticipating a need beyond this one, which is the only one that exists.  Since RDR-Export is unique in a variety of ways, it feels appropriate to special-case it here too.